GOVERNANCE.md

```diff
@@ -2,46 +2,57 @@
 
 This document defines project governance for Linkerd.
 
-## Roles
+## Contributors
 
-There are two roles that convey decision-making powers: maintainer and
-super-maintainer. MAINTAINERS.md defines the membership of these roles.
+Linkerd is for everyone. Anyone can become a Linkerd contributor simply by
+contributing to the project, whether through code, documentation, blog posts,
+community management, or other means. As with all Linkerd community members,
+contributors are expected to follow the [Linkerd Code of
+Conduct][coc].
 
-## Expectations
+All contributions to Linkerd code, documentation, or other components in the
+Linkerd GitHub org must follow the guidelines in [CONTRIBUTING.md][contrib].
+Whether these contributions are merged into the project is the prerogative of
```

Comment:
typo:
```suggestion
Whether these contributions are merged into the project is the perogative of
```

Reply:
https://github.com/linkerd/linkerd2/pull/5477#pullrequestreview-562066340 -- turns out it's actually prerogative.

Reply:
whoa, i guess i've been spelling it wrong my entire life. never mind, then!